VERSION

```diff
@@ -1 +1 @@
-0.8.0
+0.8.1
```

Comment:
I believe we'll want to bump the version in a separate PR, and do that version bump PR when we're ready to go through the release process.

This is one case where adding a CHANGELOG.md entry makes sense. This is essentially a "user facing" change, in the sense that customers are using our Golang API, and definitely need to know with which version of Golang this is compatible.

So I'd suggest:
- Undo this version number change for now.
- Add an entry in the CHANGELOG.md file for this Golang version upgrade (under `UNRELEASED` in the CHANGELOG.md for now).

After this is merged, we can do the full release process.